sdk/tests/js/tests/gl-object-get-calls.js

```diff
@@ -739,7 +739,7 @@ if (contextVersion > 1) {
     debug("Test getInternalformatParameter")
 
     gl.bindRenderbuffer(gl.RENDERBUFFER, renderbuffer);
-    wtu.glErrorShouldBe(gl, gl.NO_ERROR);
```

Comment:
Just noticed it, but can you move this out of the (contextVersion > 1) condition and put it after above deleteRenderbuffer() statement? It should be tested for WebGL1 also.